src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.cpp

```diff
@@ -240,6 +240,28 @@ Value getDimOrConstant(ConversionPatternRewriter &rewriter, Location loc,
              : create.math.constant(type, shape[axis]);
 }
 
+/// Check whether this op should be lowered to Krnl.Call according to option
+/// opsToCall. The op name is used for matching
+bool checkOpToCall(Operation *op, std::string opsForCall) {
```

Comment:
That implementation is fine by now; eventually if we have many entries, we may want to create a dictionary. If you make another pass at this PR, maybe add a comment indicating that if in the future we have many calls, we may want to revise this implementation.